iocage/lib/Distribution.py

```diff
@@ -75,6 +75,24 @@ def fetch_releases(self):
         self.available_releases = available_releases
         return available_releases
 
+    def get_release_trunk_file_url(self, release, filename):
+
+        if self.host.distribution.name == "HardenedBSD":
+            # ToDo: implement HardenedBSD release updates
+            raise Exception(
+                "Updates of HardenedBSD releases not supported yet"
+            )
+        elif self.host.distribution.name == "FreeBSD":
+
+            if release.name == "11.0-RELEASE":
```

Comment:
Nope, right after 10.0.0 there was the special release of 10.0.1 - I heard the rumor this was an exception. Do you know a better way to make sure always to grab the latest minor patches?